kura/org.eclipse.kura.util/src/main/java/org/eclipse/kura/util/service/ServiceUtil.java

```diff
@@ -61,7 +62,10 @@ private ServiceUtil() {
             final Collection<ServiceReference<T>> refs = bundleContext.getServiceReferences(clazz, filter);
             return refs.toArray(new ServiceReference[0]);
         } catch (final InvalidSyntaxException ise) {
-            throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, ThrowableUtil.stackTraceAsString(ise));
+            final StringWriter sw = new StringWriter();
+            final PrintWriter pw = new PrintWriter(sw);
+            ise.printStackTrace(pw);
```

Comment:
As I mentioned earlier, instead of using org.eclipse.kura.core.util.ThrowableUtil.stackTraceAsString or writing it all again is not necessary. You still can use the utility methods from ThrowableUtil inside Utility bundle.